### DISCUSSION TITLE
File System Corruption when hijacking

### DISCUSSION THREAD

**Discussion by username-is-already-taken2** [concourse/concourse#8170] [General]
Hi 

Just wondered if anyone has seen this before (I've been using concourse for a little while and its new to me)

![image](https://user-images.githubusercontent.com/13700297/158366910-ed779899-de93-45de-8d5b-2c010c072644.png)

My concourse version is 6.7.2, the container I'm hijacking is based on `python:3.9-slim` when the task is running it seems to be able to read and write the files fine so its only a blocker for me troubleshooting.

No worries if not, I have a docker-compose instance running concourse 7 so I'll fire up the same image in that to check

**Comment by taylorsilva** [comment]
This may be an issue with the hijacking user having a different UUID than what the container's filesystem was chown'd to. Haven't seen this issue on v7 and my memory on v6 stuff is getting really fuzzy now, so not sure what is really happening. That's my best guess.
